src/site/shared/containers/SelectionPopup/index.tsx

```diff
@@ -33,7 +33,7 @@ export function SelectionPopup({info, modules, docsUrlConfig}: Props) {
 
             // Make sure all components have same ID
             const ids = selections.get().map(GetIDFor);
-            setID((ids.length > 0 && ids.every(id => id === ids[0])) ? ids[0] : "");
+            setID(((ids.length > 0 && ids.every(id => id === ids[0])) ? ids[0] : "") ?? "");
```

Comment:
at least make it `(ids[0] ?? "")` but better to see why it thinks `ids[0]` is undefined, and can probably just be `!`'d